API/collection.py

```diff
@@ -1,89 +1,82 @@
-from typing import List, Union
+from typing import List
 
-from fastapi import Form, APIRouter, HTTPException, File, UploadFile
-from pydantic.class_validators import Optional
+from fastapi import Form, APIRouter, File, UploadFile
 from sqlalchemy import select
 from sqlalchemy.orm import Session
 
 from API.database import get_database, DB
 from Models.collection import Collection
 from Models.item import Item
 from Models.set import Set
-from Models.wardrobe import Wardrobe
+from starlette.requests import Request
 
 conn = get_database()
 database = DB(conn)
 router = APIRouter()
 
 # TODO Patch for sets and items
-@router.post("/sets")
```

Comment:
Maybe good thing is split routing to API/set.py API/item.py like with wardrobe ?

Reply:
It's a good idea, I think the item should be in item.py, but set should stay with collection